docs/build/voting-dapp-tutorial.md

```diff
@@ -0,0 +1,1245 @@
+---
+title: dApp Tutorial
+description: Build a complete dApp on Kadena
+---
+
+import PageRef from '@components/PageRef'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Complete dApp Tutorial
```

Comment:
Can we come up with a more descriptive title? Maybe something like: `First steps with Kadena` or something catchier

Reply:
From "zero to hero" with Kadena?